README.md

```diff
@@ -74,30 +74,56 @@ At the top of the script, you'll see a JavaScript dictionary of configuration se
 
 ## Custom Use
 
-You can import the monkey in an existing set of UI Automation script files and add custom events to trigger like so:
+As delivered the monkey starts itself. If you import it, it will start running immediately, which is probably not what you want if you want to customize its use. If you want to control when the monkey is released please follow the pattern in the SampleCustomization folder. In brief you want to set a global as set in SetGlobals.js, but due to Apple's javascript implementation you cannot simply set it beofre you import UIAutoMonkey.js. Instead you need to follow the pattern in the SampleCustomization folder.
 
-    // Usage & Customization example
-    // Save this UIAutoMonkey.js somewhere in your disk to import it and configure it in each of your Instruments instances
-    #import "/path/to/UIAutoMonkey.js"
+Save UIAutoMonkey.js somewhere in your disk to import it and configure it in each of your Instruments instances
+### In your test file
+
+```
+"use strict";
+#import "Includes.js"
 
-    // Configure the monkey: use the default configuration but a bit tweaked
-    monkey = new UIAutoMonkey()
-    monkey.config.numberOfEvents = 1000;
-    monkey.config.screenshotInterval = 5;
-    // Of course, you can also override the default config completely with your own, using `monkey.config = { … }` instead
 
-    // Configure some custom events if needed
-    monkey.config.eventWeights.customEvent1 = 300;
-    monkey.allEvents.customEvent1 = function() { … }
+// Usage & Customization example
 
-    // Release the monkey!
-    monkey.RELEASE_THE_MONKEY();
+// Configure the monkey: use the default configuration but a bit tweaked
+monkey = new UIAutoMonkey();
+monkey.config.numberOfEvents = 1000;
+monkey.config.screenshotInterval = 5;
+// Of course, you can also override the default config completely with your own, using `monkey.config = { … }` instead
 
-Make sure you comment out the last line of `UIAutoMonkey.js` if you include it in your project:
+// Configure some custom events if needed
+monkey.config.eventWeights.customEvent1 = 300;
+monkey.allEvents.customEvent1 = function() { … }
+
+// Release the monkey!
+monkey.RELEASE_THE_MONKEY();
+
+```
+
+###Includes.js
+Setup a single includes file. Due to Apple javascript limitations it is best to have all of your imports there. E.G
+
+```
+#import "SetGlobals.js"
+#import "/path/to/buttonHandler.js"
+"#import "/path/to/UIAutoMonkey.js""
+```
+You will need to adjust the imports to the path for UIAutoMonkey.js and the optional buttonHandler.js files
+
+###SetGlobals.js
+Have at least this global there:
+
+```
+UIAutoMonkeyClientWillReleaseTheMonkey = true;
+```
+
+When the monkey is released using the default release mechanism (UIAutoMonkeyClientWillReleaseTheMonkey is undefined or false) it executes
+```UIALogger.logDebug("Releasing the monkey directly from UIAutoMonkey");
+```
```

Comment:
Might want to double check this code snippet. It's rendering funny when I preview the markdown file. This is what I'm seeing:

![screen shot 2015-06-28 at 9 43 16 pm](https://cloud.githubusercontent.com/assets/20848/8401120/d2343446-1dde-11e5-8539-fe7b23de7824.png)